docs/changelog.rst

```diff
@@ -28,7 +31,7 @@ Fixed
 - fix error in **setup_reservoirs** when gdf contains no data in np.nanmax calculation for i.e. damheight #35
 - write_forcing with time cftime.DatetimeNoLeap #138 by removing slicing forcing if missings (not needed)
 - write_forcing automatic adjustment of starttime and endtime based on forcing content
-- When clipping a model from a model with multiple forcing files, a single netcdf is made in write_forcing and the * is removed from the filename. 
+- When clipping a model from a model with multiple forcing files, a single netcdf is made in write_forcing and the * is removed from the filename.
 
 Deprecated
 ----------
```

Comment:
Add that setup_hydrodem is removed

Reply:
Thanks for spotting this, I have now added this